packages/app/src/systems/Transaction/__mocks__/abi.ts

```diff
@@ -0,0 +1,70 @@
+export const CONTRACT_CALL_ABI = {
+  types: [
```

Comment:
Is this repeated in the other mock?  I guess we can think about centralizing the location of the abis in a later pr.